docs/governance/community-guidelines/ways-of-working.md

```diff
@@ -0,0 +1,32 @@
+# Ways of Working
```

Comment:
Great concept here. Some thoughts below.

Reply:
Thanks, replied to our comments. One question though, why are you awake right now?

Reply:
Woke up for one reason, opened github for another. 

The brain is a funny thing. 

Life is a funny thing.